modules/engage-paella-player-7/public/default_theme/theme.css

```diff
@@ -123,6 +123,30 @@ button {
   --main-outline-color: #f200f2;
 }
 
+.captions-canvas.visible-ui .text-container {
+  bottom: 65px !important;
```

Comment:
!important should not be used if it can be avoided - check if you can increase the specificity (i.e add more descriptive path)

Also - pixels are probably not the right choice for this size: have you tested it on other screen sizes and on  a mobile device ? 

https://joshcollinsworth.com/blog/never-use-px-for-font-size
https://betterprogramming.pub/stop-using-the-pixel-unit-in-css-8b8788a1301f

Reply:
`!important` is used because we're overriding styles that are already defined within the paella player plugin codebase.

And, pixels were changes to rem for font-size and percentage for the bottom property.